tox.ini

```diff
@@ -1,6 +1,7 @@
 [tox]
 envlist=
     py{35,36}-{core,p2p,database,transactions,vm,native-blockchain}
+    py{35,36}-{auxiliary, sharding}
```

Comment:
nitpick: I think this should go into the line above, no?

Reply:
yeah, I think so.  👍
(we were trying to isolate sharding stuff, but they haven't been separated in `core` anyway)
I will modify it after merging.